interfaces/builtin/network_manager.go

```diff
@@ -99,6 +101,9 @@ network packet,
 # in the same place.
 /run/NetworkManager/{,**} rw,
 
+# Allow access to the system dbus
+/run/dbus/system_bus_socket rw,
```

Comment:
We can't directly use `#include <abstractions/###DBUS_ABSTRACTION###>` here since the `####DBUS_ABSTRACTION###` placeholder will not get replaced - instead this should just be `#include <abstractions/dbus-strict>` as is done in various other interfaces that use the system dbus socket.

Reply:
Strange, since `#include <abstractions/dbus-strict>` is already a part of the connected plug